docs/apache-airflow/index.rst

```diff
@@ -15,65 +15,120 @@
-- **Extensible**:  Easily define your own operators, executors and extend the library so that it fits the level of abstraction that suits your environment.
-- **Elegant**:  Airflow pipelines are lean and explicit. Parameterizing your scripts is built into the core of Airflow using the powerful **Jinja** templating engine.
-- **Scalable**:  Airflow has a modular architecture and uses a message queue to orchestrate an arbitrary number of workers. Airflow is ready to scale to infinity.
+- A DAG named "demo", starting on Jan 1st 2022 and running once a day. A DAG is Airflow's representation of a workflow.
+- Two tasks, a BashOperator running a Bash script and a PythonOperator running a Python script
+- ``>>`` between the tasks defines a dependency and controls in which order the tasks will be executed
 
+Airflow evaluates this script and executes the tasks at the set interval and in the defined order. The status
+of the "demo" DAG is visible in the web interface:
 
-Beyond the Horizon
-------------------
+.. image:: /img/hello_world_graph_view.png
+  :alt: Demo DAG in the Graph View, showing the status of one DAG run
 
-Airflow **is not** a data streaming solution. Tasks do not move data from
-one to the other (though tasks can exchange metadata!). Airflow is not
-in the `Spark Streaming <http://spark.apache.org/streaming/>`_
-or `Storm <https://storm.apache.org/>`_ space, it is more comparable to
-`Oozie <http://oozie.apache.org/>`_ or
-`Azkaban <https://azkaban.github.io/>`_.
+This example demonstrates a simple Bash and Python script, but these tasks can run any arbitrary code. Think
+of running a Spark job, moving data between two buckets, or sending an email. The same structure can also be
+seen running over time:
+
+.. image:: /img/hello_world_grid_view.png
+  :alt: Demo DAG in the Grid View, showing the status of all DAG runs
+
+Each column represents one DAG run. These are two of the most used views in Airflow, but there are several
+other views which allow you to deep dive into the state of your workflows.
+
+Why Airflow?
+=========================================
+Airflow is a batch workflow orchestration platform. The Airflow framework contains operators to connect with
+many technologies and is easily extensible to connect with a new technology. If your workflows have a clear
+start and end, and run at regular intervals, they can be programmed as an Airflow DAG.
+
+If you prefer coding over clicking, Airflow is the tool for you. Workflows are defined as Python code which
+means:
+
+- Workflows can be stored in version control so that you can roll back to previous versions
+- Workflows can be developed by multiple people simultaneously
+- Tests can be written to validate functionality
+- Components are extensible and you can build on a wide collection of existing components
+
```

Comment:
```suggestion
- Developing a complex DAG logic or customizing an operator is as easy as using an existing operator. 
  You do not need to learn another language if existing operators are not enough -
  you continue to use Python
```

Reply:
I see where you are going with these, but I think we can do without.